packages/e2e-tests/specs/keyboard-navigable-blocks.test.js

```diff
@@ -3,121 +3,120 @@
-	);
-	await expect( isFocusedParagraphBlock ).toEqual( 'core/paragraph' );
-
-	// Tab causes 'add block' button to receive focus
-	await page.keyboard.press( 'Tab' );
-	const isFocusedParagraphInserterToggle = await page.evaluate( () =>
-		document.activeElement.classList.contains( 'block-editor-inserter__toggle' )
-	);
-	await expect( isFocusedParagraphInserterToggle ).toBe( true );
-
-	await tabThroughBlockMoverControl();
-	await tabThroughBlockToolbar();
-
-	// Tab causes the paragraph content to receive focus
-	await page.keyboard.press( 'Tab' );
-	const isFocusedParagraphContent = await page.evaluate(
-		() => document.activeElement.contentEditable
-	);
-	// The value of 'contentEditable' should be the string 'true'
-	await expect( isFocusedParagraphContent ).toBe( 'true' );
-
-	const paragraphEditableContent = await page.evaluate(
-		() => document.activeElement.innerHTML
-	);
-	await expect( paragraphEditableContent ).toBe( paragraphText );
-};
-
-const tabThroughBlockMoverControl = async () => {
-	// Tab to focus on the 'move up' control
-	await page.keyboard.press( 'Tab' );
-	const isFocusedMoveUpControl = await page.evaluate( () =>
-		document.activeElement.classList.contains( 'block-editor-block-mover__control' )
-	);
-	await expect( isFocusedMoveUpControl ).toBe( true );
-
-	// Tab to focus on the 'move down' control
-	await page.keyboard.press( 'Tab' );
-	const isFocusedMoveDownControl = await page.evaluate( () =>
-		document.activeElement.classList.contains( 'block-editor-block-mover__control' )
-	);
-	await expect( isFocusedMoveDownControl ).toBe( true );
-};
-
-const tabThroughBlockToolbar = async () => {
-	// Tab to focus on the 'block switcher' control
-	await page.keyboard.press( 'Tab' );
-	const isFocusedBlockSwitcherControl = await page.evaluate( () =>
-		document.activeElement.classList.contains(
-			'block-editor-block-switcher__toggle'
-		)
-	);
-	await expect( isFocusedBlockSwitcherControl ).toBe( true );
-
-	// Tab to focus on the 'Change text alignment' dropdown control
-	await page.keyboard.press( 'Tab' );
-	const isFocusedChangeTextAlignmentControl = await page.evaluate( () =>
-		document.activeElement.classList.contains( 'components-dropdown-menu__toggle' )
-	);
-	await expect( isFocusedChangeTextAlignmentControl ).toBe( true );
-
-	// Tab to focus on the 'More formatting' dropdown toggle
-	await page.keyboard.press( 'Tab' );
-	const isFocusedBlockSettingsDropdown = await page.evaluate( () =>
-		document.activeElement.classList.contains( 'components-dropdown-menu__toggle' )
-	);
-	await expect( isFocusedBlockSettingsDropdown ).toBe( true );
-};
-
 describe( 'Order of block keyboard navigation', () => {
 	beforeEach( async () => {
 		await createNewPost();
 	} );
 
-	it( 'permits tabbing through paragraph blocks in the expected order', async () => {
-		const paragraphBlocks = [ 'Paragraph 0', 'Paragraph 1', 'Paragraph 2' ];
+	describe( 'Order of heading block keyboard navigation', () => {
```

Comment:
Having both `describe` and `it` is redundant. They should be rather consolidated into one `it` statement. In the test output you will see at the moment:
`Order of block keyboard navigation` > `Order of heading block keyboard navigation` > `permits tabbing through heading blocks in the expected order`